prelude/prelude.purs

```diff
@@ -352,11 +422,17 @@ module Prelude
     show GT = "GT"
     show EQ = "EQ"
 
+  -- | Compare the ranking of two data of the same type.
```

Comment:
Is this documentation for `compare` or `Ord`? If the former, I again prefer "values" or "inhabitants". If the latter, I would say "define a total ordering for a type".